tests/func/test_remote.py

```diff
@@ -31,10 +31,10 @@ def test(self):
 
         self.assertEqual(main(["remote", "list"]), 0)
 
-        self.assertEqual(main(["remote", "remove", remotes[0]]), 0)
         self.assertEqual(
             main(["remote", "modify", remotes[0], "option", "value"]), 0
         )
+        self.assertEqual(main(["remote", "remove", remotes[0]]), 0)
```

Comment:
Updated the this test because it was trying to modify a setting after removing it :sweat_smile: